### PR TITLE
Update README with Uninstall Instructions

### PR DESCRIPTION
**Pull request by dmares01**
As easy as it should be for most people to uninstall the program I think adding this will reduce the number of uninstall issues popping up

### CODE REVIEW

**Comment by dmares01** [comment]
Moved to install section as referenced in issue https://github.com/Rem0o/FanControl.Releases/issues/1177

**Comment by dmares01** [comment]
Also targets https://github.com/Rem0o/FanControl.Releases/issues/1186 as well

File: README.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ JayzTwoCents - Everyone NEEDS this FREE piece of software... You will thank me!
 3. Start FanControl.exe
 4. (Optional) -c or --config [json config file] command line arg 
 
+### Uninstall Instructions
+Since __Fan Control__ is a standalone app you can not use Windows to uninstall it. That being said the uninstall process is quite simple. Navigate to where you originally extracted the files for the program. If all the files are in their own directory, feel free to delete the entire directory. If the files are not in their own directory, well you're out of luck and need to delete each file individually.
+
+If you have __Fan Control__ set to automatically start when Windows starts up you also need to delete the startup task in task scheduler.
+1) In the search bar type __Task Scheduler__ and open the app. From there you should see an image similar to the one below.
```
Comment (Rem0o) [inline README.md:48]: The image is unnecessary and takes a lot of README space.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,15 @@ JayzTwoCents - Everyone NEEDS this FREE piece of software... You will thank me!
...
+![fan_control_screenshot](https://user-images.githubusercontent.com/33143771/196514548-19097cab-a3f1-447a-8a94-778c749d73f2.png)
+2) Find the __Fan Control__ task in the list, right click on it, and select _delete_.
```
Comment (Rem0o) [inline README.md:50]: "3)" is redundant.

**Review by Rem0o — COMMENTED** [review]
I expected much less. I feel like this can be summarize in basically 2 sentences. 

The 2 points are as fellow: 
1) FanControl is a stand-alone app, there is nothing to uninstall, EXCEPT IF...
2) ... you have start with windows, uncheck the checkbox, or manually delete the "FanControl" task in the Task Scheduler.

File: README.md (hunk 1 above)
Comment (dmares01) [reply README.md:48]: Sounds good that can all get pulled out then

**Pull request merged**